core-services/prow/03_deployment/boskos.yaml

```diff
@@ -146,11 +146,11 @@ objects:
     namespace: ${namespace}
   spec:
     port:
-      targetPort: 8082
+      targetPort: 8080
     to:
       kind: Service
-      name: boskos
+      name: boskos-redirect
     wildcardPolicy: None
     tls:
       insecureEdgeTerminationPolicy: Redirect
-      termination: Reencrypt
+      termination: edge
```

Comment:
This is btw the reason it didn't work initially, as the redirecting nginx does not have a tls listener :)